Kudu.Core/Functions/KedaFunctionTriggerProvider.cs

```diff
@@ -70,23 +73,56 @@ bool IsHostJson(string fullName)
             return triggers;
         }
 
-        public static IEnumerable<ScaleTrigger> GetFunctionTriggers(IEnumerable<JObject> functionsJson, string hostJsonText)
+        internal static void updateFunctionTriggerBindingExpression(
+            IEnumerable<ScaleTrigger> scaleTriggers, IDictionary<string, string> appSettings)
+        {
+            string ReplaceMatchedBindingExpression(Match match)
```

Comment:
nit: move this to static helper to add unit tests?

Reply:
UpdateFunctionTriggerBindingExpression already has an unit test. Is it not enough?

Reply:
I also noticed that this project using netcoreapp2.2 that is out of support.  The unit test suddenly stop working. I had to install netcoreapp 2.2. Just FYI